modules/ROOT/pages/usage-reports.adoc

```diff
@@ -122,14 +122,11 @@ You must redeploy your apps with a supported version, either manually or during
 
 To xref:cloudhub-2::ch2-update-apps.adoc#roll-back-to-a-previous-successful-configuration[roll back] to a previous successful configuration of existing applications that use a Mule version that usage reports do not support, choose a compliant version before deploying changes. After deployment, MuleSoft creates a new configuration with the previously chosen settings and a supported runtime.
 
-== Enable Access to Usage Reports
-
-To grant user access to usage reports, an organization administrator must grant the Usage Viewer permission.
-
 == Access Usage Reports
 
-To access usage reports, you must have the Usage Viewer permission in Access Management.
+To access usage reports, you need the xref:access-management::permissions-by-product.adoc#usage[Usage Viewer permission]. If you do not have the Usage Viewer permission, contact your organization administrator to request access. The Usage Viewer permission must be xref:access-management::managing-permissions.adoc[applied at the root organization level] rather than the business group level.
```

Comment:
```suggestion
To access usage reports, you need the xref:access-management::permissions-by-product.adoc#usage[Usage Viewer permission]. If you do not have the Usage Viewer permission, contact your organization administrator to request access. The Usage Viewer permission must be xref:access-management::managing-permissions.adoc[applied at the root organization level] rather than the business group level. Usage reports are available only when you are accessing the root organization and do not appear at the business group level.
```